greenheart/tools/eco/hydrogen_mgmt.py

```diff
@@ -250,15 +342,17 @@ def run_h2_storage(
 
         h2_storage = None
 
-    elif design_scenario["h2_storage_location"] == "turbine":
-        if greenheart_config["h2_storage"]["type"] == "turbine":
+    elif greenheart_config["h2_storage"]["type"] == "turbine":
```

Comment:
Want to consider a different paradigm for this elif structure, thinking along the lines of separate solver files for each type.